plugins/CoreHome/VisitorDetails.php

```diff
@@ -17,17 +17,24 @@ class VisitorDetails extends VisitorDetailsAbstract
     public function extendVisitorDetails(&$visitor)
     {
         $visitor['userId']                      = $this->getUserId();
-        $visitor['visitorType']                 = $this->getVisitorReturning();
-        $visitor['visitorTypeIcon']             = $this->getVisitorReturningIcon();
+        if ($this->isDataProfilable) {
+            $visitor['visitorType'] = $this->getVisitorReturning();
```

Comment:
does this maybe impact the logic in `plugins/Live/templates/_visitorDetails.twig`?

Reply:
should be resolved